scripts/pyinfra-complete.sh

```diff
@@ -1,21 +1,27 @@
-_pyinfra_completion() {
-    local IFS=$'
-'
-    COMPREPLY=( $( env COMP_WORDS="${COMP_WORDS[*]}" \
-                   COMP_CWORD=$COMP_CWORD \
-                   _PYINFRA_COMPLETE=complete $1 ) )
-    return 0
-}
+#!/usr/bin/env bash
```

Comment:
What's the reasoning with changing this script? The old/current one is dynamic so will always be up to date whereas this hardcodes the options?

Reply:
It didn't work for me when I tried it earlier. I'll re-test.

Reply:
Ok. It works. Not sure what I did last time.

Reply:
Will close this PR and open a new one.